src/raygun.utilities/index.js

```diff
@@ -211,18 +211,6 @@ window.raygunUtilityFactory = function(window, Raygun) {
       return Math.floor(Math.random() * 9007199254740993);
     },
 
-    getViewPort: function() {
-      if (this.isReactNative()) {
-        return { width: 'Not available', height: 'Not available' };
```

Comment:
Here the null object is defined like this with strings, but in the newer `getViewportDimensions` function we return nulls. Will this have negative impacts if the worker was relying on this "Not available" string for some reason?

Reply:
Good point, I decided to get rid of this as we want to move away from having workarounds for RN and migrate RN users to the new provider. I'll check if this value is important.

Reply:
I can't find any references to the "Not available" string, so the only change here would be that these values would now display as null in Crash Reporting

Reply:
Yep, that would be a good way to handle this - this is something we can do when we update the infrastructure :+1: